src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -51,7 +51,7 @@ namespace System.Management.Automation
     ///     - There is no ITypeDescriptorContext.
```

Comment:
@iSazonov, your last commit had 5 failures in `PowerShell-CI-windows`
Language Primitive Tests.Test convertion with .Net Core intrinsic type convertor
```powershell
Exception calling "ConvertTo" with "2" argument(s): "Cannot convert the "2,3" value of type "System.String" to type "System.Drawing.Point"."
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Api\LanguagePrimitive.Tests.ps1: line 32
32:         $result = [System.Management.Automation.LanguagePrimitives]::ConvertTo('2,3', [System.Drawing.Point])

```
Remoting loopback tests.Can connect to all disconnected sessions by name
```powershell
Cannot bind parameter 'ResourceURI'. Cannot convert value "Shell" to type "System.Uri". Error: "Invalid URI: The format of the URI could not be determined."
at <ScriptBlock>, D:\a\1\s\test\tools\Modules\HelpersRemoting\HelpersRemoting.psm1: line 249
249:     Connect-PSSession @parameters

```
New-ModuleManifest tests.Relative URIs are not allowed
```powershell
Cannot convert value "../foo" to type "System.Uri". Error: "Invalid URI: The format of the URI could not be determined."
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Module\NewModuleManifest.Tests.ps1: line 115
115:         $testUri = [Uri]"../foo"

```
WSMan Config Provider.New-Item and Remove-Item tests.New-Item and Remove-Item for a plugin using parameters
```powershell
Cannot bind parameter 'Resource'. Cannot convert value "foo" to type "System.Uri". Error: "Invalid URI: The format of the URI could not be determined."
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.WSMan.Management\ConfigProvider.Tests.ps1: line 338
338:                     -SDKVersion 2 -Resource foo -Capability shell -XMLRenderingType text -Path WSMan:\localhost\plugin `

```
ConvertTo-Html Tests.Test ConvertTo-Html using page parameters
```powershell
Cannot bind parameter 'CssUri'. Cannot convert value "page.css" to type "System.Uri". Error: "Invalid URI: The format of the URI could not be determined."
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\ConvertTo-Html.Tests.ps1: line 83
83:         $returnString = ($customObject | ConvertTo-Html -Title "Custom Object" -Body "Body Text" -CssUri "page.css" -As List) -join $newLine

```